packages/element/README.md

```diff
@@ -225,6 +225,16 @@ _Related_
 
 -   <https://reactjs.org/docs/react-api.html#reactmemo>
 
+<a name="Platform" href="#Platform">#</a> **Platform**
+
+Component used to detect the current Platform being used. Use Platform.OS === 'web' to detect if running on web enviroment.
+This is the same concept as the React-Native implementation found here: <https://facebook.github.io/react-native/docs/platform-specific-code#platform-module>
```

Comment:
I think it's fine to link to the react-native docs, but I'd prefer to document explicitly, since we only support `OS` and `select` on the web.